front/src/app/LogsTable.tsx

```diff
@@ -62,3 +68,28 @@ export default function LogsTable({ logs }: Props) {
         </CardAtom>
     );
 }
+
+const ClickableColHeader = ({
+    title,
+    sortKey,
+}: {
+    title: string;
+    sortKey: "datetime_utc" | "total_tokens" | "id";
+}) => {
+    const { searchParams, updateSearchParam } = useNavigation();
+    const sortBy = (key: keyof llm_logs) => {
+        const order = searchParams.get("sortOrder") === "asc" ? "desc" : "asc";
```

Comment:
Would it be good to have a string union type e.g. type sortOrderType = "asc" | "desc"

Reply:
It is defined line 11 of data/helpers/formatSearchParamToObject